Changes needed to get doxygen to configure and build with the GNU compilers
on Solaris.

--- doxygen-1.8.9/doc/Doxyfile.orig	2015-03-02 18:48:16.796824973 -0800
+++ doxygen-1.8.9/doc/Doxyfile	2015-03-02 18:47:58.404324590 -0800
@@ -23,7 +23,7 @@
 EXTRACT_ALL       = NO
 EXTRACT_PRIVATE   = NO
 GENERATE_MAN      = NO
-GENERATE_LATEX    = YES
+GENERATE_LATEX    = NO
 GENERATE_HTML     = YES
 GENERATE_HTMLHELP = NO
 GENERATE_RTF      = NO
